setup-r-dependencies/README.md

```diff
@@ -247,6 +247,21 @@ without a formal version requirement, then add the package to the
 `extra-packages` parameter. `setup-r-dependencies` installs the latest versions
 of these packages.
 
+# Using private packages from GitHub
+
+Since GitHub Actions won't be able to able to access any internal package repositories (e.g., [JFrog Artifactory](https://jfrog.com/artifactory/)), 
+you can install internal packages from GitHub directly.
+
+1. Add repo name in the `Repos: mygithuborg/myinternalpackage` to your `DESCRIPTION` file. AFAIK, the `myinternalpackage` repo location must match the name of the package.
```

Comment:
Hmmmm, I am not sure how `Repos` works. Maybe `Remotes`?

Reply:
Oh yeah, should be remotes. Good catch!!

Reply:
Fixed.